stats.rb

```diff
@@ -80,98 +48,86 @@ def tablecreation(title,pr_array)
 total_merged_pulls = 0
 total_mentioned_pulls = 0
 
-  puts "repo, last comment, needs rebase, fails test, needs squash, no comments, total open, has mention, no activty 40 days"
-repos.each do |repo|
-  pr_information_cache = util.fetch_async("#{options[:namespace]}/#{repo}", search_with={:state=>'open', :sort=>'updated'}, filter=[:statuses, :pull_request_commits, :issue_comments, :pull_request])
+puts 'repo, last comment, needs rebase, fails test, needs squash, no comments, total open, has mention, no activty 40 days'
```

Comment:
Minor enhancement:
```
puts 'repo, last comment, needs rebase, fails test, needs squash, no comments, total open, has mention, no activty 40 days' unless parsed.empty?
```
It will prevent the headings being printed out if there's no content to iterate over.